python/scripts/mass_nifti_pic.py

```diff
@@ -102,16 +102,25 @@ def input_error_checking(profile, smallest_id, largest_id, usage):
         print(usage)
         sys.exit(lib.exitcode.MISSING_ARG)
 
+    if os.path.isfile(profile):
+        sys.path.append(os.path.dirname(profile))
+        config_file = __import__(os.path.basename(profile[:-3]))
+    else:
+        message = f'\n\tERROR: you must specify a valid profile file.\n{profile} does not exist!'
+        print(message)
+        print(usage)
+        sys.exit(lib.exitcode.INVALID_PATH)
+
     if not smallest_id:
-        message = '\n\tERROR: you must specify a smallest PhysiologyFileID on ' \
-                  'which to run the chunking script using -s or --smallest_id option'
+        message = '\n\tERROR: you must specify a smallest FileID on which to run the' \
+                  ' mass_nifti_pic.py script using -s or --smallest_id option'
```

Comment:
fascinating how the error message was wrong for 6 years and my eyes never caught it every time I tested the script... 😄